config/etc/system-image/writable-paths

```diff
@@ -13,6 +13,7 @@
 /tmp                                    none                    temporary   none        defaults
 /mnt                                    none                    temporary   none        defaults
 /media                                  none                    temporary   none        defaults
+/boot/androidboot                       auto                    persistent  none        none
```

Comment:
The usual configuration for bootloaders:

snap_mode
snap_try_core
snap_try_kernel

We need a place that is accessible from both bootlader and core that is read/write for these variables.